lib/reed_solomon_decode_bb_impl.cc

```diff
@@ -115,7 +115,7 @@ namespace gr {
     void
     reed_solomon_decode_bb_impl::forecast(int noutput_items, gr_vector_int &ninput_items_required)
     {
-      ninput_items_required[0] = noutput_items;
+      ninput_items_required[0] = noutput_items*120/110;
```

Comment:
Yes, but in this case noutput_items is set to a multiple of 110, and therefore (noutput_items*120) is also a multiple of 110, so we get an integer result.